### DISCUSSION TITLE
Audible degradation of sound quality

### DISCUSSION THREAD

**Discussion by nnlkcncff** [apohl79/audiogridder#829] [Q&A] [answered]
Has anyone noticed a degradation in sound quality when using a plugin through the AudioGridder on a remote node compared to using the same plugin on a local node in a DAW in the usual way?

I can clearly hear the difference when playing guitar.
Perhaps I need a second high-quality audio interface, but the audio stream does not pass through A/D-converters between network nodes, which means that the quality drops before transmission over the network.
On the official site I did not find any information about quality degradation, on the contrary — it said "32/64 bit float processing".

For a complete understanding of the situation, I drew 2 diagrams for comparison:

![AudioGridder](https://user-images.githubusercontent.com/82151266/143686538-2f4f94dc-ce6a-4a52-b782-db0ea3892ccb.png)

**Comment by apohl79** [comment, marked as answer]
fyi https://github.com/apohl79/audiogridder/issues/302#issuecomment-754785110

**Reply by nnlkcncff** [reply]
Thanks for the answer.
Now, knowing that AudioGridder doesn't do the processing by itself, I checked everything again and found my mistake — there was a 0.8dB difference in tuning on the output signal in the plugin.

But I found another problem. I will create a new discussion, this one can be closed.
